README.md

```diff
@@ -263,6 +263,31 @@ issue. Note that you'll have to set these up yourself, since using
 [time-literals](https://github.com/henryw374/time-literals) is not an option in
 this case, see the next point.
 
+### Deja-fu with Transit
+You may want to use [Transit](https://github.com/cognitect/transit-js) to send deja-fu times over the wire in your JSON payloads, such as the out-of-the-box behavior with ShadowCLJS. Here is an example of how this is done for goog.date.Dates, where the [above table](#types) can be used to swap in other time types you may be using. 
+
+```clojure
+(ns deja-foo
+  (:require [cognitect.transit :as transit]
+            [lambdaisland.deja-fu :as fu]
+            [goog.date.Date]))
+
+(let [auth-write-handler {:handlers {goog.date.Date
+                                       (transit/write-handler
+                                        (constantly "LocalDate")
+                                        fu/format)}}
+        auth-read-handlers  {:handlers {"LocalDate"
+                                        (transit/read-handler 
+                                         fu/parse-local-date)}}
+        treader (transit/reader. "json" auth-read-handlers)
+        twriter (transit/writer. "json" auth-write-handler) ]
```

Comment:
I don't think the periods are necessary here? `"json"` can be a keyword instead: `:json`